src/components/Home/HomeNavigation/index.js

```diff
@@ -211,7 +211,7 @@ function HomeNavigation() {
                     icon={icon}
                     path={location.pathname}
                     description={description}
-                    isDiscouraged={isDiscouraged}
+                    isDiscouraged={isDiscouraged === 'true'}
```

Comment:
@SzymonZiebaNeti can you comment on this change by Michal? I'm not sure how is this related to this PR.